src/components/schedule-page/table/schedule/schedule.component.tsx

```diff
@@ -12,13 +10,7 @@ import { TableMiddleLine } from "../../table-middleline";
 import { OvertimeHeaderComponent } from "../../../overtime-header-table/overtime-header.component";
 
 export function ScheduleComponent(): JSX.Element {
-  const scheduleModel = useSelector((state: ApplicationStateModel) => state.scheduleData.present);
-
-  const { scheduleLogic, scheduleLocalState, setNewSchedule } = useScheduleState(scheduleModel);
-
-  useEffect(() => {
-    setNewSchedule(scheduleModel);
-  }, [scheduleModel, setNewSchedule]);
+  const { schedule: scheduleLocalState } = useContext(ScheduleLogicContext);
```

Comment:
To jest inna składnia do dekonstrukcji obiektów. 
`{ key: newVariableName }`
Używam tego tutaj, bo w związku z tym, że wyniosłem context wyżej, musiałem też rozszerzyć o dodatkowe ( właśnie to pod którym piszę ten komentarz ) pole